CHANGELOG.md

```diff
@@ -6,11 +6,34 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.6.0] - 2020-05-04
+
 ### Added
 - Introduce `SECRETLESS_HTTP_CA_BUNDLE` environment variable, which provides a
   path to a bundle of CA certificates to append to the certificate pool used 
-  for verification by all http service connectors
-  [#1180](https://github.com/cyberark/secretless-broker/pull/1180)
+  for server certification verification by all http service connectors.
+  [PR #1180](https://github.com/cyberark/secretless-broker/pull/1180)
+- MSSQL supports TLS for the Secretless-to-server connection, configured using
+  the `sslmode` Secretless configuration variable which supports values:
+  `disable`, `require`, `verify-ca`, and `verify-full`. With support for
+  `sslmode=verify-full`, support is also added for an `sslhost` configuration
+  parameter that Secretless will compare to the server certificate Subject
+  Alternate Names, to verify the server certificate hostname is as expected.
+  [#1163](https://github.com/cyberark/secretless-broker/issues/1163),
+  [#1164](https://github.com/cyberark/secretless-broker/issues/1164),
+  [#1165](https://github.com/cyberark/secretless-broker/issues/1165),
+  [#1199](https://github.com/cyberark/secretless-broker/issues/1199),
+  [Epic #1032](https://github.com/cyberark/secretless-broker/issues/1032)
```

Comment:
Overly verbose entries for a changelog

Reply:
do you have a suggestion for a better way to do it? I agree, there is a lot there - but that's also a little because there are many issues in the one line that I found it tricky to split out

Reply:
```
### Added
- Introduce `SECRETLESS_HTTP_CA_BUNDLE` environment variable for appending
  http service connector validation certificates.
  [PR #1180](https://github.com/cyberark/secretless-broker/pull/1180)
- Add TLS support to MSSQL connector backend connection.
- Add support for `sslhost` configuration to MSSQL connector backend connection.
  [#1163](https://github.com/cyberark/secretless-broker/issues/1163),
  [#1164](https://github.com/cyberark/secretless-broker/issues/1164),
  [#1165](https://github.com/cyberark/secretless-broker/issues/1165),
  [#1199](https://github.com/cyberark/secretless-broker/issues/1199),
```

You'd just have to split the MSSQL issues between the last two bullet points -it doesn't have to be 100% exact. Link to epic isn't needed.

Reply:
not bad - I'd like to get @alexkalish's take too

Reply:
I like separating larger entries if they are separate.  However, I feel like @sgnn7's examples above are a little terse.  There always need to be some value expressed by line.  They should provide some type of answer to "why should I care about this change?"

So, maybe more like:
```
- Introduce `SECRETLESS_HTTP_CA_BUNDLE` environment variable for providing HTTP service validation
    certificates via the environment, instead of the existing .... <blah, blah, blah>.
- Add TLS support to the MS-SQL connector for the backend connection between Secretless Broker and
    the database.  This is the recommended way to secure this connection and achieves feature
    parity with other connectors.
```

Reply:
We just need to be careful that we comply w/ keepachangelog format though - some of this stuff is getting machine read in the suite.

> However, I feel like @sgnn7's examples above are a little terse. There always need to be some value expressed by line. 

I don't necessarily disagree but changelogs have been historically the place where you want 0 clutter and the tersest of phrasings. If the reader wants more info, they are expected to follow the links at the ends. E.g. https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.9.md

Reply:
Interesting perspective.  I'm not opposed to keeping changelogs very terse, if that is how we want to roll, but it places more importance on every issue or PR doing a good job explaining why someone would care about a given change.